src/tools/rest-backend/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_package (CppCMS)
+find_package (Boost 1.50 COMPONENTS system filesystem thread REQUIRED)
```

Comment:
I have to create temporary files to use with the storage plugins for conversion. I can have another look at this and see if it works without boost too. I think I used it because boost offers methods to prevent creating already existing temp files.